pkg/health/client/client.go

```diff
@@ -345,20 +332,53 @@ func formatNodeStatus(w io.Writer, node *models.NodeStatus, printAll, succinct,
 				formatPathStatus(w, "Secondary", addr, "      ", verbose)
 			}
 		}
+		return true
 	}
+
+	if !nodeIsHealthy(node) {
+		ips := []string{getPrimaryAddressIP(node)}
+		for _, addr := range GetHostSecondaryAddresses(node) {
```

Comment:
We should dry this up with something like printNode() so we can reuse the call for unhealthy and all nodes.

Reply:
Good idea. Implemented.